waspc/src/Wasp/Util.hs

```diff
@@ -57,6 +58,13 @@ camelToKebabCase camel@(camelHead : camelTail) = kebabHead : kebabTail
         (zip camel camelTail)
     isCamelHump (a, b) = (not . isUpper) a && isUpper b
 
+projectNameToAppIdentifier :: String -> String
+projectNameToAppIdentifier = concat . capitalizeWords . wordsBy (== '-')
+  where
+    capitalizeWords :: [String] -> [String]
```

Comment:
```suggestion
    capitalizeAllWordsExceptForTheFirstOne :: [String] -> [String]
```

Reply:
I know it is a bit silly as to how long it is, but it is just to easy to understand now :D!

Reply:
updated :slightly_smiling_face: